Packs/MalwareInvestigationAndResponse/README.md

```diff
@@ -22,7 +22,7 @@ The pack closely maps evidence to MITRE ATT&CK to uncover evidence of:
 * Supports aggregating incidents from EDRs, including multiple alerts and files.
 * Supports fetching malware incidents either from the EDR product or from a SIEM solution.
 
```

Comment:
lets add a line here for the account enrichment as well

Reply:
Done